### PR TITLE
ceylon: require Java 8 specifically

### PR DESCRIPTION
**Pull request by apjanke**
- [X] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Fixes run time failure under Java 10.
Addresses #28760

### CODE REVIEW

File: Formula/ceylon.rb [hunk 1]
```diff
@@ -3,19 +3,20 @@ class Ceylon < Formula
...
   def install
     man1.install Dir["doc/man/man1/*"]
     doc.install Dir["doc/*"]
     libexec.install Dir["*"]
-
-    # Symlink shell scripts but not *.plugin
-    bin.install_symlink "#{libexec}/bin/ceylon"
-    bin.install_symlink "#{libexec}/bin/ceylon-sh-setup"
+    bin.mkdir
+    mv "#{libexec}/bin/ceylon", bin
```
Comment (DomT4) [inline Formula/ceylon.rb]: If you use `bin.install` instead of `mv` it should handle the `mkdir` part for you?

Comment (apjanke) [reply Formula/ceylon.rb]: Amended.

Comment (DomT4) [reply Formula/ceylon.rb]: Thanks! ❤️

**Comment by ilovezfs** [comment]
Perfect! Thanks @apjanke

**Pull request merged**